src/assets/scss/application.scss

```diff
@@ -99,3 +99,25 @@
 .govuk-btn-link:hover {
   background: none;
 }
+
+// Mostly copied from F2F CRI styling https://github.com/govuk-one-login/ipv-cri-f2f-front/blob/8c05ac2b93cbe92113de5877f29b4101cc4e2b15/src/assets/scss/application.scss#L129
```

Comment:
Ideally we wouldn't have to do this, but there doesn't seem to be a cross-identity set of shared styles and this isn't a GOV.UK Design System component.